nntrainer/src/neuralnet.cpp

```diff
@@ -1021,24 +1022,21 @@ std::shared_ptr<Layer> NeuralNetwork::_make_act_layer(ActiType act) {
       std::make_shared<ActivationLayer>();
 
     act_layer->setActivation(act);
+    act_layer->setInputDimension(prev->getOutputDimension());
```

Comment:
In Layer, if ActiType == ACT_UNKNOW, then there will be error. But there is an layer which does not need activation such as flatten, pooling. I think we have to consider it.

``` int Layer::setActivation(ActiType acti) {
  int status = ML_ERROR_NONE;
  if (acti == ACT_UNKNOWN) {
    ml_loge("Error:have to specify activation function");
    return ML_ERROR_INVALID_PARAMETER;
  }
  activation_type = acti;

  return status;
}```